regression/scripts/VTOL/mission_QuadShot.py

```diff
@@ -0,0 +1,550 @@
+# vtol_UAV.py
+# 
+# Created:  Jan 2016, E. Botero
+# Modified: 
+
+#----------------------------------------------------------------------
+#   Imports
```

Comment:
i dont think is called in the automatic regression script?

Reply:
This is temporarily removed. Will be included in the VTOL branch.

Reply:
The Weights Branch will pulled in prematurely without the VTOL networks. As a result the quickfix to create the old pull request do not work (without the VTOL branch)